config/jobs/etcd-druid/etcd-druid-api-unit-tests.yaml

```
@@ -17,7 +17,7 @@ presubmits:
         containers:
           # Run all tests sequentially in one container or as separate prow jobs.
           # Test will fail when 'check-generate' (which includes a revendor) and 'test' or 'check' run in parallel on the same volume
-          - image: europe-docker.pkg.dev/gardener-project/releases/ci-infra/golang-test:v20250318-9342bff-1.23
+          - image: europe-docker.pkg.dev/gardener-project/releases/ci-infra/golang-test:v20250317-d044b63-1.24
```

Please keep the original hash and only change the version, same for other places.
```suggestion
          - image: europe-docker.pkg.dev/gardener-project/releases/ci-infra/golang-test:v20250318-9342bff-1.24
```

---

addressed in https://github.com/gardener/ci-infra/pull/3410/commits/355cbbf86a084341ed26f706172d282952b7acfb